ovos_tts_plugin_piper/espeak_wrapper.py

```
@@ -0,0 +1,378 @@
+            ValueError: If the language code is unsupported.
+        """
+        if target_lang.lower() == "en-gb":
+            return "en-gb-x-rp"
+        if target_lang in cls.ESPEAK_LANGS:
+            return target_lang
+        if target_lang.lower().split("-")[0] in cls.ESPEAK_LANGS:
+            return target_lang.lower().split("-")[0]
+        return cls.match_lang(target_lang, cls.ESPEAK_LANGS)
+
+    @staticmethod
+    def _run_espeak_command(args: List[str], input_text: str = None, check: bool = True) -> str:
```

Ruff (0.12.2)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)